components/ContactEmail.tsx

```diff
@@ -0,0 +1,23 @@
+'use client';
+
+import { Obfuscate } from '@south-paw/react-obfuscate-ts';
+import { ReactElement } from 'react';
+
+interface ContactEmailProps {
+    email: string;
+}
+
+export default function ContactEmail({
+    email,
+}: ContactEmailProps): ReactElement {
+    return (
+        <Obfuscate
```

Comment:
What does this `Obfuscate` do?

Reply:
Reverse the text until a user interacts with it (eg. hover over it)

More details here: https://github.com/South-Paw/react-obfuscate-ts